scripts/aws/lambda/TM-Extractor/dev/env.hcl

```diff
@@ -0,0 +1,13 @@
+# Set common variables for the environment. This is automatically pulled in in the root terragrunt.hcl configuration to
+# feed forward to the child modules.
+
+locals {
```

Comment:
Should these be set with example defaults as a `env.hcl.example` with README instructions to replace and rename?

Reply:
I think it would be up to us to have it already created. I have updated `README.md` within `scripts/aws/lambda/TM-Extrator/` with this information. 
@dakotabenjamin However i had no clue on what stag/prod input vars will be for `account_name`, `account_region`. Can you have that updated for stag/prod.